### ISSUE TITLE
JKSV newest version doesnt open

### ISSUE THREAD

**Issue by llaci1997** [J-D-K/JKSV#162]
TLDR: when i open a newest version of JKSV theres a 1-2 sec black screen and it drop me back to a homebrew menu im on 15.0.0 and AMS 1.4.0|S

i want to backup my pokemon arceus save file with an older version of jksv but it create a 1kb size file which is empty so i update jksv for the newest version after that it work perfectly i open it and backup my save i do a little pkhexing on the save file and i reupload to my switch and i want to restore the new save so i try to open jksv but doesnt open just drop back to the homebrew i put back the older version of the jksv and the app is start but not work (not backup or restore the save it is because of the 15.0.0)
I tried it in applet mode and not in applet mode as well but same behavior.
any idea?

**Comment by J-D-K** [comment]
Try deleting the config file and let me know of that solves it. I was still trying to figure out why it causes this randomly. If it doesn't, I'll get back to you.
